libraries/AP_Math/AP_Math.cpp

```diff
@@ -510,3 +510,12 @@ uint32_t float_to_uint32(const float v)
 {
     return uint32_t(constrain_float(v, 0, UINT32_MAX));
 }
+
+
+// set the value of a Vector3f and set as default
+void set_and_default_vector3f(AP_Vector3f &pv, const Vector3f &v)
```

Comment:
Does this work? I though we only only got a single float default for vectors. Or at least that is all we get in the the tables.

Reply:
it works by providing the value to this function

Reply:
Sorry, what I meant was is there any conflict between somewhere trying to set the defaults of all values and this trying to set them individually. I think the param pointer for the first element would be the same as that for the whole thing. If I were to do a `set_default_by_name` for each element would it still work as expected? I don't think we ever try and set a default for the whole thing, its just there is a single value in the table, so think its OK.